docs/source/markdown/podman-create.1.md

```diff
@@ -672,11 +678,16 @@ Set the network mode for the container. Invalid if using **--dns**, **--dns-opt*
 
 Valid _mode_ values are:
 
-- **bridge**: Create a network stack on the default bridge. This is the default for rootfull containers.
+- **bridge[:OPTIONS,...]**: Create a network stack on the default bridge. This is the default for rootfull containers. It is possible to specify these additional options:
```

Comment:
It would be good to have an example of one of these below, especially the `IP` or the `mac` as we talk about them earlier.